vscode/src/testController.ts

```diff
@@ -677,8 +681,8 @@ export class TestController {
       const end = item.range.end;
 
       testItem.range = new vscode.Range(
-        new vscode.Position(start.line, start.column),
-        new vscode.Position(end.line, end.column),
+        new vscode.Position(start.line, start.character),
```

Comment:
Should these lines also change?
https://github.com/Shopify/ruby-lsp/pull/3236/files#diff-4970e950e007c1318a00bbe05a2e39b92ab1466ebff44ed740a8953a02dc6a4dL144-L145

Reply:
No, because that code is the old implementation that doesn't use the same request.